.github/workflows/go.yml

```diff
@@ -7,16 +7,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ 1.17.x, 1.18.x ]
+        go: [ 1.17.x, 1.18.x, 1.19.x ]
```

Comment:
@GiddyGoatGaming Could you remove `1.17.x` at the same time please? We only need to test the two most recent supported versions.

Reply:
> @GiddyGoatGaming Could you remove `1.17.x` at the same time please? We only need to test the two most recent supported versions.

Done!